player/lua/osc.lua

```diff
@@ -2585,7 +2585,11 @@ function tick()
 
         -- render idle message
         msg.trace("idle message")
-        local icon_x, icon_y = 320 - 26, 140
+        local _, _, display_aspect = mp.get_osd_size()
+        local display_h = 360
+        local display_w = display_h * display_aspect
+        -- logo is rendered at 2^(6-1) = 32 times resolution with size 1800x1800
+        local icon_x, icon_y = (display_w - 1800 / 32) / 2, 140
```

Comment:
This changes the location, because 1800/64 is ~28, not 26. Was the previous location incorrect?

Reply:
It was not perfectly centered (I checked in a screenshot)